test_runner/src/main/kotlin/ftl/args/yml/YamlErrorConverter.kt

```diff
@@ -13,3 +14,8 @@ fun convertConfigurationErrorExceptions(missingParameterError: Exception, yaml:
         missingParameterError
     }
 }
+
+fun convertConfigurationErrorExceptions(treeParsingException: MarkedYAMLException): Throwable {
+    val errorMessageBuilder = ConfigurationErrorMessageBuilder
```

Comment:
If `ConfigurationErrorMessageBuilder.invoke` will be a top level function, this assignment wouldn't be needed ;).

Reply:
I would like to have it consistent with previous message builder, however I move it to single line

Reply:
to make this consistent I leave this in ConfigurationErrorMessageBuilder, but changed to a single line and avoid assignment